wp-content/themes/redmond/web/scss/layout/_footer.scss

```diff
@@ -1,3 +1,56 @@
 .footer {
-
+  background-color: $secondary-color;
+  padding: rem-calc(36px) rem-calc(80px);
+  color: $primary-light-color;
+  h2 {
+    color: $primary-light-color;
+    font-family: 'Lato', sans-serif;
+    font-size: rem-calc(19px);
+    font-weight: normal;
+    & small {
+      color: $primary-light-color;
+      font-style: normal;
+      text-transform: uppercase;
+      font-size: rem-calc(16px);
+    }
+  }
+  .footer-icons {
+    border-bottom: 1px #b8bfc2 solid;
```

Comment:
make this a var in settings. call it footer-icons-border or something like that.